.pylintrc

```diff
@@ -82,3 +90,14 @@ disable=
 	# if:/else: and for:/else:.
 	useless-else-on-loop,
 	no-else-return,
+	# NOTE(lidiz): Python 3 make object inheritance default, but not PY2
+	useless-object-inheritance,
+	# NOTE(lidiz): Limit the length of function doesn't mean reducing
+	# complexity.
+	too-many-statements,
```

Comment:
How many places does this fail? Not immediately clear that this should be disabled globally versus locally as needed

Reply:
I think I'm with @ericgribkoff  on this one. Even if it should be caught by code review, it's nice to have a linter as a safety net. If we consciously decide to make an exception, adding an exception comment isn't very burdensome.

Reply:
The failure for this check looks like it's for `_consume_request_iterator` in `_channel.py`, and that is a *long* method - let's leave this enabled globally, as it would be nice not to have more methods of similar size getting introduced